frontend/src/components/ListOfMedications.jsx

```diff
@@ -1,20 +1,15 @@
 import MedicationCard from "../Patient/PatientComponents/PatientDashboardPageComponents/MedicationCard.jsx";
 import MedicationToApproveCard from "../Provider/ProviderComponents/MedicationToApproveCard.jsx"
 import { Fragment, useEffect, useState } from "react";
-import { getPatientMedication, getProviderMedicationsToApprove } from "../../testAPI.js";
+import { getPatientMedications } from "../api.js";
```

Comment:
[non-blocking] You can avoid having to repeatedly update imports for API by importing things in a named object
```suggestion
import * as API from "../api.js";
```

Then you can use this as `API. getPatientMedications(id)`.  This will help signify when a function call is an API call vs a normal react hook / function.